runner/tests/imports-missing/sub/BUILD.out

```diff
@@ -1,3 +1,11 @@
 load("@deps-test//my:rules.bzl", "x_lib")
 
-x_lib(name = "b")
+x_lib(
```

Comment:
This demonstrates what we are trying to fix...

Note this `.out` file is now the same as [the .in file](https://github.com/aspect-build/aspect-gazelle/blob/62739a4814eb15440754bdfa7be88b92ccf92028/runner/tests/imports-missing/sub/BUILD.in) because when a resolution error occurs we now output to stderr and then halt. Previously it would spam stdout and then continue running and updated the BUILD files which was destructive in this case.